tests/integration-tests/tests/common/utils.py

```diff
@@ -40,21 +46,43 @@
     "ubuntu1804": {"name": "Deep Learning Base AMI (Ubuntu 18.04)*", "owners": ["amazon"]},
 }
 
-AMI_TYPE_DICT = {"official": OS_TO_OFFICIAL_AMI_NAME_OWNER_MAP, "remarkable": OS_TO_REMARKABLE_AMI_NAME_OWNER_MAP}
+OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP = {
+    "alinux": {"name": "amzn-hvm-x86_64-*", "owners": ["amazon"]},
```

Comment:
Is it possible to have multiple owner for the same image? 
If it's not the case I'd rename it to "owner" and convert it into a string in place of a list.

Reply:
Yes, e.g. 
```
    "ubuntu1604": {
        "name": "ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-*-server-*",
        "owners": ["099720109477", "513442679011", "837727238323"],
    },
```

`Owners` is the same parameter passed to the `describe_images` API